csharp/Ice/callback/Client.cs

```diff
@@ -17,12 +17,12 @@ public static int Main(string[] args)
             // The new communicator is automatically destroyed (disposed) at the end of the
             // using statement
             //
-            using(var communicator = Ice.Util.initialize(ref args, "config.client"))
+            using (var communicator = Ice.Util.initialize(ref args, "config.client"))
```

Comment:
We should switch to using statements.

Reply:
Sure, that's unrelated to this PR however.